packages/agoric-cli/tools/getting-started.js

```diff
@@ -105,121 +102,31 @@ export const gettingStartedWorkflowTest = async (t, options = {}) => {
       initOptions.push(...opts);
     }
     t.is(
-      await myMain([
-        'init',
-        '--dapp-template',
-        'dapp-fungible-faucet',
-        ...initOptions,
-        'dapp-foo',
-      ]),
+      await myMain(['init', ...initOptions, 'dapp-foo']),
```

Comment:
As of last Christmas, [getting started](https://docs.agoric.com/guides/getting-started/) no longer uses the `agoric` CLI.

This step should probably be `yarn create @agoric/dapp --dapp-template dapp-offer-up dapp-foo`